config/samples/nsm_v1alpha1_nsm.yaml

```diff
@@ -8,6 +8,9 @@ spec:
   nsmPullPolicy: IfNotPresent
   
   # Forwarding Plane Configs
-  forwardingPlaneName: vpp
-  forwardingPlaneImage: ghcr.io/networkservicemesh/cmd-forwarder-vpp
+  forwardingPlanes:
+    - Name: vpp
+      Image: ghcr.io/networkservicemesh/cmd-forwarder-vpp:v1.1.0
+    - Name: vlan
```

Comment:
@LionelJouin Could you get me some pointers to those issues?  Is someone working on fixing them?

Reply:
Since the vlan mechanism is part of the vpp-forwarder we are no longer using the vlan-forwarder, and I am not sure if anyone is maintaining it. When we uplifted our project to NSM 1.2, I heard about issues with the vlan-forwarder, but I don't have more information.
Probably @ljkiraly or @zolug might know more about its status.

Reply:
Hi! The standalone vlan-forwarder had been abandoned, because of the decision to move towards vpp-forwarder integration. (It basically got stuck on NSM 1.0 (maybe even pre 1.0). So not surprisingly it failed to interwork with NSM 1.1)

Reply:
@LionelJouin and friends thanks for the heads up on the vlan status. This is just the sample and can totally live without that. Just the VPP forwarder is enough. I'll fix that as well.

Reply:
@LionelJouin the vlan example has been removed now.